source/description.rst

```diff
@@ -614,26 +614,26 @@ data objects. Reference ``dtypes`` are described via a dictionary. E.g.:
 .. code-block:: yaml
 
   dtype:
-        target_type: ElectrodeGroup
-        reftype: object
+    target_type: ElectrodeGroup
+    reftype: object
 
 
 ``target_type`` here describes the ``data_type`` of the target that the reference points to and
-``reftype`` describes the kind of reference. Currently the specification language supports two main
-reference types.
+``reftype`` describes the kind of reference. Currently the specification language supports one
+type of reference: object references.
 
 +--------------------------+-------------------------------------+
 | ``reftype`` **value**    | **Reference type description**      |
 +--------------------------+-------------------------------------+
-| * "ref"                  | Reference to another group or       |
-| * "reference"            | dataset of the given                |
-| * "object"               | ``target_type``                     |
-+--------------------------+-------------------------------------+
-| * region                 | Reference to a region (i.e. subset) |
-|                          | of another dataset of the given     |
+| * "object"               | Reference to another group or       |
+|                          | dataset of the given                |
```

Comment:
The question is whether the `reftype` key is still needed since all references are for objects now.

Reply:
I think it's fine to remove it. We should just make sure that it is being handled gracefully in HDMF, i.e., if it is present and the value is ``object`` then it should be ignored and if ``reftype`` is being set to anything else but ``object`` then HDMF should raise a warning.

Reply:
Oops. I merged without thinking... I will create another PR with this change.